### DISCUSSION TITLE
[求助]这是什么地方？有人知道怎么回去吗？

### DISCUSSION THREAD

**Discussion by KELERSAMA** [share121/inter-knot#2187] [其他]

**Comment by yutong903** [comment]
这个委托我接了
先找那个蓝色头发的女孩，说你自己要加入他们，就你后面那个，然后就摆烂吧
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by KELERSAMA** [reply]
哦哦，那个蓝毛刚跟一个火红的法师走。等下次见到的时候问问。
